force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -226,9 +251,9 @@ public with sharing class VariablesDataTypesOperators {
      */
     public static Double complexOrderOfOperations(Integer a, Integer b, Integer c, Integer d, Integer e) {
         // Add parentheses around the multiplication and subtraction operations so that they are performed before division and addition
-        Double result = a * b - c / (Double) d + e;
+        Double result = (((a * b - c) / (Double) d) + e);
```

Comment:
You don't need the outer parentheses wrapped around the entire expression. The parentheses around `a * b` are also not necessary in this case because the order of operations ensures multiplication will happen before subtraction.